networking/aws_load_balancer_operator/aws-load-balancer-operator-release-notes.adoc

```diff
@@ -68,6 +68,12 @@ The following advisory is available for the AWS Load Balancer Operator version 1
 
 * link:https://access.redhat.com/errata/RHEA-2023:1954[RHEA-2023:1954 Release of AWS Load Balancer Operator on OperatorHub Enhancement Advisory Update]
 
+[IMPORTANT]
+====
+AWS Load Balancer Operator version 1 cannot upgrade automatically from version 0 (the version available for tech preview).
```

Comment:
```suggestion
The AWS Load Balancer Operator version 1.0.0 cannot upgrade automatically from version 0.x.x, the Technology Preview version.
```

Reply:
Can the user manually upgrade the Operator? 

> The AWS Load Balancer Operator version 1.0.0 cannot upgrade from version 0.x.x.

Reply:
ty